src/widgets/Ruler.cpp

```diff
@@ -67,6 +52,11 @@ array of Ruler::Label.
 #include "Theme.h"
 #include "ViewInfo.h"
 
+#include "Updater.h"
+// Need to include to set default
+#include "LinearUpdater.h"
+#include "LogarithmicUpdater.h"
```

Comment:
The commit that separates RulerPanel.cpp can also remove this `#include` which becomes unnecessary.